index.html

```diff
@@ -0,0 +1,75 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Startup Matchmaker</title>
+    <link rel="stylesheet" type="text/css" href="css/styles.css">
+  </head>
+  
+  <body>
+    <div class="top-section">
+      <header>Startup Matchmaker</header>
+      
+      <nav>
+        <a class="nav-find" href="#">Find Developers</a>
+        <a class="nav-find" href="#">Find Designers</a>
+        <a href="#">How it Works</a>
+        <a href="#">Our Team</a>
+        <a href="#">Blog</a>
+      </nav>
+
+      <hr>
+    </div>
+
+    <div class="tag-line">
+      <h1>Because two brains <strike>taste</strike> <u>are</u> better than one.</h1>
```

Comment:
Try to not to presentational tags for styling, defer to using css. It give you much more flexibility and control over what you styling. To make this match the a mare gineric tag like `<span>`. Spans are link inline `div`. The n give that a border-bottom 🔨